docs/features/pm2-api.md

```diff
@@ -36,22 +34,120 @@ pm2.connect(function(err) {
     instances : 4,                // Optional: Scale your app by 4
     max_memory_restart : '100M'   // Optional: Restart your app if it reaches 100Mo
   }, function(err, apps) {
-    pm2.disconnect();
+    pm2.disconnect();   // Disconnect from PM2
+    if (err) {
+      throw new Error('err');
+    }
   });
 });
 ```
 
-## No daemon
-
-Make sure you killed pm2 before via `pm2 kill`, then to start pm2 programmatically in no deamon mode, you need to pass the `true` as the first argument of `pm2.connect`:
-
-```javascript
-var pm2 = require('pm2');
+## Programmatic API
 
-pm2.connect(true, function(err) {
-  // Do stuff
-});
-```
+`npm install pm2 --save`
+
+**`pm2.connect(errback)`** - Connects to a running pm2 daemon ("God") or launches and daemonizes one. If launched, the pm2 process will continue running after the script exits.  
+**`pm2.connect(noDaemonMode, errback)`**
+
+* `noDaemonMode` - (Default: false) If true is passed for the first argument, pm2 will not be run as a daemon and will die when the script starting it exits. By deafult, pm2 is stays alive after your script exits. Make sure pm2 isn't already running when you start it in no-daemon mode.  
```

Comment:
> By deafult, pm2 is stays alive after your script exits. Make sure pm2 isn't already running when you start it in no-daemon mode.  

By default, pm2 stays alive after your script exits. If pm2 is already running, your script will attach the existing daemon but will also die when your process exits.